src/routez/router.zig

```diff
@@ -302,7 +306,7 @@ pub fn match(
         }
     };
     const r = pathbuf[begin..index];
-    if (!mem.eql(u8, r, path[path_index..])) {
+    if (!mem.eql(u8, r, path[path_index..]) and route.path[0] != '*') {
```

Comment:
Put this in the other order so that the if will not be emitted for global paths.